jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/security/authentication/SslClientCertAuthenticator.java

```diff
@@ -70,15 +71,20 @@ public Authentication validateRequest(Request req, Response res, Callback callba
             Response.writeError(req, res, callback, HttpServletResponse.SC_FORBIDDEN);
             return Authentication.SEND_FAILURE;
         }
-        
+
+        if (sslSessionData.cipherSuite() != null)
+        {
+            req.setAttribute("jakarta.servlet.request.cipher_suite", sslSessionData.cipherSuite());
+        }
```

Comment:
This is wrong, as the `cipherSuite` has already been set as a request attribute by the `SecureRequestCustomizer`.
Why doing this here?

Reply:
I'm not quite to understand. https://github.com/eclipse/jetty.project/blob/jetty-12.0.x/jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/SecureRequestCustomizer.java is in jetty-core so this class doesn’t have to set any servlet attribute (as could be javax or jakarta).

Reply:
I think we need to write EE secure customizers.